geoapi/services/point_cloud.py

```diff
@@ -31,7 +34,7 @@ def get(database_session, pointCloudId: int) -> PointCloud:
         :param pointCloudId: int
         :return: PointCloud
         """
-        point_cloud = database_session.query(PointCloud).get(pointCloudId)
+        point_cloud = database_session.get(PointCloud, pointCloudId)
```

Comment:
https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#orm-query-get-method-moves-to-session